packages/windows/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+-version: "1.44.2"
+  changes:
+    -  description: Added error.message and http.request.body.bytes ECS field mapping.
+       type: bugfix
+       link: https://github.com/elastic/integrations/pull/8975
```

It seems  you forgot to update the PR link
```suggestion
       link: https://github.com/elastic/integrations/pull/8976
```